source/MonoGame.Extended.Input/InputListeners/TouchEventArgs.cs

```diff
@@ -13,7 +13,7 @@ public TouchEventArgs(ViewportAdapter viewportAdapter, TimeSpan time, TouchLocat
             RawTouchLocation = location;
             Time = time;
             Position = viewportAdapter?.PointToScreen((int)location.Position.X, (int)location.Position.Y)
-                ?? location.Position.ToPoint();
+                ?? new Point((int)location.Position.X, (int)location.Position.Y);
```

Comment:
Vector2.ToPoint() is an MG extension. It doesn't exist in FNA.
An alternative is to add an extension method like Nez does.

Reply:
Done.